pytorch_lightning/trainer/progress.py

```diff
@@ -213,18 +234,17 @@ def reset_on_epoch(self) -> None:
     def load_state_dict(self, state_dict: dict) -> None:
         self.optimizer.load_state_dict(state_dict["optimizer"])
         self.scheduler.load_state_dict(state_dict["scheduler"])
+        self.optimizer_idx = state_dict["optimizer_idx"]
 
 
 @dataclass
 class EpochLoopProgress(BaseProgress):
     """
     Tracks epoch loop progress.
     These counters are local to a trainer rank. By default, they are not globally synced across all ranks.
-
```

Comment:
Can you configure your linter to not remove these newlines? The `Args:` section should be separated